src/app/services/BD/index.ts

```diff
@@ -29,7 +34,8 @@ export const getSelectedOffice = async (id: string) => {
     .get()
     .then((snapshot) => {
       if (snapshot.exists()) {
-        return snapshot.val();
+        const selectedOfficeData = snapshot.val();
+        return OfficeDataUpdated(selectedOfficeData);
```

Comment:
`OfficeDataUpdated` sounds line variable name. Better to use verbs for functions: `prepareOfficeData`